### DISCUSSION TITLE
Resource class deprecation

### DISCUSSION THREAD

**Discussion by sagikazarmark** [distribution/distribution#4060] [Q&A]
Hey folks!

I was told on the OCI Slack that the resource class in access scopes is actually deprecated/ignored now.

I was trying to find some reference to back that up and to add it to some documentation as a reference, but couldn't find any.

Simply by just looking at the code, it's not obvious that it's been deprecated or is being ignored.

There is also no mention of that in the documentation: https://github.com/distribution/distribution/blob/main/docs/spec/auth/scope.md#resource-class

Would it make sense to update the documentation if it is indeed not used anymore? (To be clear: I'm volunteering to send a PR if that's the case)

**Comment by milosgajdos** [comment]
Yes, please feel free to update the documentation. We'll be spending a bit more time on it in the runup to v3 release.
